packages/transport/src/sessions/background.ts

```diff
@@ -294,6 +313,12 @@ export class SessionsBackground extends TypedEmitter<{
         };
     }
 
+    private getInternal(pathPublic: PathPublic): PathInternal | undefined {
+        return Object.entries(this.pathInternalPathPublicMap).find(
+            ([_internal, pub]) => pub === pathPublic,
+        )?.[0];
```

Comment:
Maybe a bit better readable?
```suggestion
        return Object.keys(this.pathInternalPathPublicMap).find(
            internal => this.pathInternalPathPublicMap[internal] === pathPublic,
        );
```